janitor/functions/select.py

```diff
@@ -312,6 +343,11 @@ def select(
         is with `.loc` or `.iloc` methods, as they are generally performant.
         `select` is primarily for convenience.
 
+    !!! abstract "Version Changed"
+
+        - 0.26.0
+            - Added variable `args`, `invert` and `axis` parameters.
```

Comment:
Is it worth mentioning that `rows` is deprecated in favour of `index` here in the docstring?